src/qt/spinbutt.cpp

```diff
@@ -34,12 +35,32 @@ void wxQtSpinButton::valueChanged(int value)
     wxSpinButton *handler = GetHandler();
     if ( handler )
     {
-        wxCommandEvent event( wxEVT_SPIN, handler->GetId() );
+        wxSpinEvent event( wxEVT_SPIN, handler->GetId() );
         event.SetInt( value );
         EmitEvent( event );
     }
 }
 
+void wxQtSpinButton::stepBy(int steps)
+{
+    wxSpinButton* handler = GetHandler();
```

Comment:
Really nitpicky, but:
```suggestion
    wxSpinButton* const handler = GetHandler();
```